lms/djangoapps/courseware/views/views.py

```diff
@@ -761,7 +769,7 @@ def _progress(request, course_key, student_id):
     return response
 
 
-def _get_cert_data(student, course, course_key, is_active, enrollment_mode):
+def _get_cert_data(student, course, course_key, is_active, enrollment_mode, national_id):
```

Comment:
@pepeportela Do not break API function call. You can get national_id inside the `_get_cert_data` function with the `student` variable (User).